test/integration/environment/.env

```diff
@@ -1,6 +1,6 @@
 POSTGRES_VERSION=13
 DATAVERSE_DB_USER=dataverse
 SOLR_VERSION=9.3.0
-DATAVERSE_IMAGE_REGISTRY=docker.io
-DATAVERSE_IMAGE_TAG=unstable
+DATAVERSE_IMAGE_REGISTRY=ghcr.io
+DATAVERSE_IMAGE_TAG=10216-metadatablocks-payload-extension
```

Comment:
This will be reverted once https://github.com/IQSS/dataverse/pull/10225 is merged into dataverse develop. For now we can test the PR with this PR-generated image. Before merging this PR it is important to revert these values.

Reply:
Reverted, since https://github.com/IQSS/dataverse/pull/10225 is merged.